APE0.rst

```diff
@@ -296,15 +300,18 @@ The procedure for voting to add new Voting Members is:
 * The Coordination Committee and Ombudsperson receive each nomination, check
   that it is factually accurate, that the nominated person accepts the
   nomination, and that their record of community activity adheres to the Code of
-  Conduct.
-* Once a nomination is accepted by the Coordination Committee, the Coordination
-  Committee will put it to a vote among active Voting Members. Votes for adding
-  voting members will be carried out at most twice a year to avoid too many
-  elections. The voting dates will be announced with at least four weeks notice,
+  Conduct. If the candidate accepts, they should provide a brief statement in
+  support of their nomination.
+* Once a nomination is accepted by the Coordination Committee, it will be put it
+  to a vote among active Voting Members. Votes for adding voting members will be
+  carried out at most twice a year to avoid too many elections. The returns
+  officer for the voting member election can be a voting member, and those
+  officer are entitled to vote in the election.
```

Comment:
```suggestion
  officer(s) for the election of the voting members can be a voting member, and those
  officer(s) are entitled to vote in the election.
```

Reply:
@mwcraig @adrn Is the slight grammar tweak good to merge?

Reply:
I'm inclined to say no since it isn't quite what the voting members voted for.

Reply:
Since there has been significant discussion beyond just grammatical issues after voting had started, should all those be moved to a follow-up PR then? Possibly with a tag indicating that the PR is frozen before voting starts, and to be merged as is including spelling errors and everything.

Reply:
Yes -- I think the next step is to merge this PR (which passed) and open a new PR for discussion then a vote along with the next CoCo election, which needs to be held sometime in the next few months.

That PR would include at least  a proposed change in the minimum number of participants needed to elect someone (to raise it from 4) and this grammatical fix.

Our next CoCo meeting will be this week or next, I will add it to the agenda.